### PR TITLE
Update embedded forms behaviour

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Update embedded forms behaviour

#### Ticket Link
None

#### Related PRs:
Webapp: https://github.com/mattermost/mattermost-webapp/pull/6804

### CODE REVIEW

File: server/http/helloapp/f_survey_embedded.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	call.Context = &apps.Context{
+		AppID: AppID,
+	}
+	return &model.Post{
+		Message: "Fill this survey",
+		Props: model.StringInterface{
+			"form": form,
+			"call": call,
+		},
+	}
```
Comment (mickmister) [inline server/http/helloapp/f_survey_embedded.go:69]: :+1:

**Review by mickmister — APPROVED** [review]
LGTM!

**Comment by mickmister** [comment]
Can we add additional fields to the helloapp survey form to include a button field and a submit field?

**Comment by larkox** [comment]
@mickmister Slight changes were made in the webapp to finish tying the dots. You can see them in this PR: https://github.com/mattermost/mattermost-webapp/pull/6804

Also, the current approach is more than welcomed to be discussed.

**Review by mickmister — APPROVED** [review]
LGTM :+1:

**Comment by larkox** [comment]
The effort in this PR is mostly the "UpdatePost" response, which will be removed. Closing this PR.

**Pull request closed without merging**